src/main/kotlin/calculator/Calculator.kt

```diff
@@ -0,0 +1,36 @@
+package calculator
+
+class Calculator(private val input: String) {
+
+    private fun isNegative(value: Int) = value < 0
+
+    private fun tokens(): List<String> {
+        val result = Regex("//(.)\n(.*)").find(input)
+        result?.let {
+            val delimiter = it.groupValues[1]
+            return it.groupValues[2].split(delimiter)
+        } ?: run {
+            return input.split(":", ",")
+        }
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        return result?.let {
            val delimiter = it.groupValues[1]
            it.groupValues[2].split(delimiter)
        } ?: input.split(":", ",")
```